openedx_webhooks/info.py

```diff
@@ -97,6 +101,19 @@ def is_bot_pull_request(pull_request):
     """
     return pull_request["user"]["type"] == "Bot"
 
+def is_no_jira_org_pull_request(pull_request):
```

Comment:
I think it's easier to reason about things expressed positively. This could be `should_add_jira_for_pull_request`, or `is_jira_pull_request` (No need for org in the name, since the caller doesn't need to know what info is being used to decide to do jira or not.)